src/main/app/src/app/zaken/zaken.service.ts

```diff
@@ -105,7 +105,7 @@ export class ZakenService {
     toekennen(zaak: Zaak, reden?: string): Observable<Zaak> {
         const toekennenGegevens: ZaakToekennenGegevens = new ZaakToekennenGegevens();
         toekennenGegevens.zaakUUID = zaak.uuid;
-        toekennenGegevens.behandelaarGebruikersnaam = zaak.behandelaar?.id;
+        toekennenGegevens.behandelaarGebruikersnaam = zaak.toekenning.medewerker?.id;
```

Comment:
En die toekenning is hier nooit null? Maar waarom check je dat elders dan wel? Ik zou dat niveau toekking weer weghalen. Of je moet duidelijk kunnen uitleggen wat het functioneel betekent. Wel/geen toekenning met wel/geen medewerker en groep. Dat zijn vijf mogelijke combinaties. Hebben die alle vijf functioneel betekenis? Dan is de extra complexiteit nodig, anders kun je je code een stuk leesbaarder maken door dat subobjectje niet te gebruiken.

Reply:
Toekenning subobject verwijderd, zie ook vorige comments. Opgelost in nieuwe commit.